proto/movementlabs/protocol_units/da/light_node/v1beta2.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+package movementlabs.protocol_units.da.light_node.v1beta2;
```

Comment:
Do we need an entirely new version of the protocol?

Reply:
It has been discussed here: https://github.com/movementlabsxyz/movement/pull/996#pullrequestreview-2559597210
And we go to a new version of the protocol.